examples-crate/uv_mapping/main.rs

```diff
@@ -18,23 +13,22 @@ enum AppState {
 
 const UV_SCALE: f32 = 1.0 / 16.0;
 
-struct Loading(Handle<Texture>);
+struct Loading(Handle<Image>);
 
 fn main() {
-    App::build()
+    App::new()
         .add_plugins(DefaultPlugins)
         .insert_resource(State::new(AppState::Loading))
         .add_state(AppState::Loading)
-        .add_system_set(SystemSet::on_enter(AppState::Loading).with_system(load_assets.system()))
-        .add_system_set(SystemSet::on_update(AppState::Loading).with_system(check_loaded.system()))
-        .add_system_set(SystemSet::on_enter(AppState::Run).with_system(setup.system()))
-        .add_system_set(
-            SystemSet::on_update(AppState::Run).with_system(camera_rotation_system.system()),
-        )
+        .add_system_set(SystemSet::on_enter(AppState::Loading).with_system(load_assets))
+        .add_system_set(SystemSet::on_update(AppState::Loading).with_system(check_loaded))
+        .add_system_set(SystemSet::on_enter(AppState::Run).with_system(setup))
+        .add_system_set(SystemSet::on_update(AppState::Run).with_system(camera_rotation_system))
         .run();
 }
 
 fn load_assets(mut commands: Commands, asset_server: Res<AssetServer>) {
+    debug!("load");
```

Comment:
From `bevy::prelude::*`?

Reply:
yes. sorry i left those in there, whoops